examples/computecloth/computecloth.cpp

```diff
@@ -77,7 +76,7 @@ class VulkanExample : public VulkanExampleBase
 		} semaphores;
 		VkQueue queue{ VK_NULL_HANDLE };
 		VkCommandPool commandPool{ VK_NULL_HANDLE };
-		std::array<VkCommandBuffer, 2> commandBuffers{};
+		VkCommandBuffer commandBuffer{ VK_NULL_HANDLE };
```

Comment:
I _think_ I get why this was two command buffers.  You were trying to set it up so that at the end of each compute loop, you took whichever SSBO had been written last and used it as the input to the next cycle?  But even though you transition both SSBOs to the graphics queue, the input buffer is _never_ used as a vertex buffer in the original code, and there aren't two corresponding graphics command buffers / descriptor sets to switch between using storageBuffer.input and storageBuffer.output as an input like there are in compute.